hack/kustomization/whereabouts/kustomization.yaml

```diff
@@ -6,6 +6,3 @@ resources:
 - ./daemonset-install.yaml
 - ./whereabouts.cni.cncf.io_ippools.yaml
 - ./whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml
-
-transformers:
```

Comment:
I think that the files appear above are deleted as well ?
(they were merged into the current file)
also once there isn't a transformers we basically not need this file and the references of it

On the other hand, we will need on a follow PR to use kustomize again in order to pin the version
until https://github.com/k8snetworkplumbingwg/whereabouts/issues/252 is fixed